documentation/modules/new-migrating-virtual-machines-cli.adoc

```diff
@@ -574,8 +574,7 @@ spec:
 EOF
 ----
 <1> Allowed values are `pod` and `multus`.
-<2> Specify a network attachment definition for each additional {virt} network. Specify the
-`namespace` either by using the `namespace property` or with a name built as follows: `<network_namespace>/<network_name>`.
+<2> Specify a network attachment definition for each additional {virt} network. Specify the `namespace` either by using the `namespace property` or with a name as follows: `<network_namespace>/<network_name>`.
```

Comment:
yes, we do allow both, but the `<namespace>/<name>` syntax will be deprecated in favor of the more user friendly `namespace: <namespace>; name: <name>`